docs/spec.md

```diff
@@ -226,6 +227,18 @@ to that of the derivation (or derivations).
 
 
 
+## Contravariant
+
+#### Methods
+
+  1. `contramap :: Contravariant f => (a → b, f b) → f a`
```

Comment:
It should be `contramap :: Contravariant f => Type f ~> (a → b, f b) → f a`.

This notations probably doesn't make much sense to anyone but me, and I [don't like it too much either](https://github.com/rpominov/static-land/issues/34). But for now we should use it in all methods for consistency.